dataflow/src/main/java/org/checkerframework/dataflow/analysis/AbstractAnalysis.java

```diff
@@ -200,6 +200,23 @@ public AnalysisResult<V, S> getResult() {
             assert !n.isLValue() : "Did not expect an lvalue, but got " + n;
             if (!currentNode.getOperands().contains(n)
                     && !currentNode.getTransitiveOperands().contains(n)) {
+                // Special handling for the postfix operation, as the operation introduces a
+                // temporary variable for CFG but not for the AST side,
+                // thus cause inconsistency.
+                // See https://github.com/eisop/checker-framework/pull/304
+                if (n instanceof AssignmentNode) {
+                    Tree.Kind nKind = ((AssignmentNode) n).getTree().getKind();
+                    if (nKind != Tree.Kind.POSTFIX_DECREMENT
+                            && nKind != Tree.Kind.POSTFIX_INCREMENT) {
+                        return null;
+                    }
+                    Node expr = ((AssignmentNode) n).getExpression();
```

Comment:
Can you add a one-sentence summary of what this does?
So it's clear here that there is some mis-match for post-fix ops, but not clear what is done about it.

Reply:
be more specific about it.